models/bebop/base/bebop_base_trades.sql

```diff
@@ -0,0 +1,51 @@
+{{ config(
+        schema = 'bebop_base',
+        alias = 'trades',
+        post_hook='{{ expose_spells(\'["base"]\',
+                        "project",
+                        "bebop",
+                        \'["alekss"]\') }}'
```

Comment:
We can use the new format for post_hook here:
```suggestion
        materialized = 'view',
        post_hook='{{ expose_spells(blockchains = \'["base"]\',
                                    spell_type = "project",
                                    spell_name = "bebop",
                                    contributors = \'["alekss"]\') }}'
```